mscore/mixertrackchannel.cpp

```diff
@@ -161,13 +161,13 @@ void MixerTrackChannel::updateNameLabel()
                            "Channel: %3\n"
                            "Bank: %4\n"
                            "Program: %5\n"
-                           "Patch: %6")
+                           "Sound: %6")
```

Comment:
Maybe better leave it "Patch:" here, as here it really is the technical term?

Reply:
Then again this here is the toolTip shown at the bottom, when hovering over the name fule below the volume slider, in ths same dialog as the one where now "Sound:" is shown at the top, where the sound can get selected, better those terms should match.